src/Admin/AfterActivationSiteScan.php

```diff
@@ -197,6 +235,11 @@ protected function add_preload_rest_paths() {
 				[ 'author', 'name', 'plugin', 'status', 'version' ],
 				'/wp/v2/plugins'
 			),
+			add_query_arg(
+				'_fields',
+				[ 'author', 'name', 'status', 'stylesheet', 'version' ],
```

Comment:
Per above, this will also need to include `template` for the sake of child themes:

```suggestion
				[ 'author', 'name', 'status', 'stylesheet', 'template', 'version' ],
```

Reply:
I've just committed that but I'm now wondering if we actually need to know the parent theme (i.e. the `template`)? We don't show an information like this anywhere in the UI so I guess we could just skip that.

Reply:
I think you'd need it here potentially:

```js
const themeMeta = themes.find( ( theme ) => theme.stylesheet === themeWithAmpIncompatibility.slug );
const themeName = themeMeta?.name?.rendered ?? themeMeta?.name;
```

Reply:
We are now using `template`, too. As noted in https://github.com/ampproject/amp-wp/pull/6859#issuecomment-1059806234, the parent/child themes were not handled well in the Site Scan results. This should be fixed with 11da989.